packages/ag-charts-enterprise/src/features/zoom/zoomAxisDragger.ts

```diff
@@ -57,7 +57,7 @@ export class ZoomAxisDragger {
             return newZoom.y;
         }
 
-        // Scale the zoom along the given axis, pivoting on the end of the axis
+        // Scale the zoom along the given axis, anchoring on the end of the axis
```

Comment:
Axis zoom is a separate ticket which I'll work on after this to make it obey the anchors.